src/sentry/logging/handlers.py

```diff
@@ -67,7 +67,7 @@ def __call__(self, logger, name, event_dict):
 class StructLogHandler(logging.StreamHandler):
     def get_log_kwargs(self, record, logger):
         kwargs = {k: v for k, v in vars(record).items() if k not in throwaways and v is not None}
-        kwargs.update({"level": record.levelno, "event": record.msg})
+        kwargs.update({"level": record.levelno, "event": record.msg, "severity": record.levelname})
```

Comment:
I think we'll have to do some mapping to get this exactly how we want: google expects uppercase, and may have slightly different names. can see how they map them here: https://github.com/googleapis/python-logging/blob/28d141d0e8ed4560d2e33f8de0d43b0825a7f33f/google/cloud/logging_v2/_helpers.py#L34

Reply:
uppercasing may be sufficient, since we don't use the extra ones google provides.

Reply:
running through PDB indicated that `levelname` was already uppercase. But i'll add a `.upper()`, to be safe

As for other log levels, yeah we don't use anything beyond the basic log levels.

Reply:
ahh i see in the docs now. yeah this should work as is then!